lib/pure/unicode.nim

```diff
@@ -1352,6 +1352,142 @@ proc isCombining*(c: Rune): bool {.rtl, extern: "nuc$1", procvar.} =
     (c >= 0x20d0 and c <= 0x20ff) or
     (c >= 0xfe20 and c <= 0xfe2f))
 
+template copyRuneToResult(runeIndex, rune) =
+  var runeStr = $rune
```

Comment:
Is there a piece of code or reference I can look at for inlining $rune to save the alloc?